arduino-hal/src/lib.rs

```diff
@@ -12,6 +12,7 @@ compile_error!(
     * arduino-nano
     * arduino-uno
     * sparkfun-promicro
+    * trinket-pro
```

Comment:
I did some documentation changes today, please rebase on the latest version of `next` and also add `trinket-pro` to the doc-attr-mess above this:

```rust
#![cfg_attr(feature = "trinket-pro", doc = "**Trinket Pro**.")]
```